src/coord/src/catalog.rs

```diff
@@ -661,23 +739,29 @@ impl CatalogItem {
             | CatalogItem::Sink(_)
             | CatalogItem::Type(_)
             | CatalogItem::Secret(_) => Err(SqlCatalogError::InvalidDependency {
-                name: name.to_string(),
+                name: name.item.clone(),
```

Comment:
Unfortunately this was the only error (that I'm aware of) that wasn't able to retain a fully qualified object name in the message. I don't think it's possible to resolve the full name from here, because we don't even have a reference to the catalog.